storage/local/storage.go

```diff
@@ -254,6 +255,12 @@ func NewMemorySeriesStorage(o *MemorySeriesStorageOptions) *MemorySeriesStorage
 			Name:      "memory_series",
 			Help:      "The current number of series in memory.",
 		}),
+		numHeadChunks: prometheus.NewGauge(prometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "num_head_chunks_open",
```

Comment:
Just "open_head_chunks" is more consistent with the naming of the other metrics.